aten/src/ATen/core/dispatch/DispatchTable.h

```diff
@@ -33,9 +33,7 @@ namespace c10 {
 class DispatchTable final {
  public:
   DispatchTable(const FunctionSchema& schema)
-  : kernels_()
-  , catchallKernel_(c10::nullopt)
```

Comment:
let's keep those initializers, but possibly remove the `c10::nullopt` (i.e. call default constructor instead) since it's not an optional anymore.